datalad/distribution/tests/test_create.py

```diff
@@ -23,6 +23,7 @@
 from datalad.tests.utils import assert_not_in
 from datalad.tests.utils import assert_in
 from datalad.tests.utils import assert_raises
+from datalad.tests.utils import assert_equal
```

Comment:
Evil nose introduced chaos... Eventually after less of outstanding PRs we should imho converge on one or another